code/game/objects/random/weapons.dm

```diff
@@ -62,29 +29,28 @@
 	icon_state = "45-10"
 
 /obj/random/ammo/item_to_spawn()
-	return pickweight(list(/obj/item/ammo_magazine/pulse = 2,
-	/obj/item/ammo_magazine/pulse/hv = 0.3,
-	/obj/item/ammo_magazine/pulse/df = 0.3,
-	/obj/item/weapon/cell/plasmacutter = 1.5,
-	/obj/item/ammo_magazine/sawblades = 1,
-	///obj/item/ammo_magazine/bullpup = 1,
-	/obj/item/weapon/cell/force = 0.75,
-	/obj/item/ammo_magazine/seeker = 1,
-	/obj/item/weapon/cell/contact = 0.75,
-	/obj/item/ammo_magazine/divet = 1.5,
-	/obj/item/ammo_magazine/lineracks = 1,
-	/obj/item/ammo_casing/tripmine = 1,
-	/obj/item/weapon/reagent_containers/glass/fuel_tank/fuel = 1,
+	return pickweight(list(/obj/item/ammo_magazine/pulse = 1.8,
+	/obj/item/ammo_magazine/pulse/hv = 0.27,
+	/obj/item/ammo_magazine/pulse/df = 0.27,
+	/obj/item/weapon/cell/plasmacutter = 1.35,
+	/obj/item/ammo_magazine/sawblades = 0.9,
+	/obj/item/weapon/cell/force = 0.67,
+	/obj/item/ammo_magazine/seeker = 0.9,
+	/obj/item/weapon/cell/contact = 0.67,
+	/obj/item/ammo_magazine/divet = 1.35,
+	/obj/item/ammo_magazine/lineracks = 0.9,
+	/obj/item/ammo_casing/tripmine = 0.9,
+	/obj/item/weapon/reagent_containers/glass/fuel_tank/fuel = 0.9,
```

Comment:
Note, lowering ALL weights by 10% doesn't actually do anything. This doesn't affect the chance of ammo spawning, merely affects which type of ammo to spawn. You would have to lower the weight of the random subtype that actually spawns random ammo spawns in maint.

Reply:
This is basically what happens when a blanket request is given for *all*, and I do mean all, ammotypes to be dropped 10%.

Reply:
... what?